jetbrains-core/src/software/aws/toolkits/jetbrains/services/cloudformation/stack/Updater.kt

```diff
@@ -134,8 +135,14 @@ class Updater(
             updating.set(!newStackStatusNotInProgress)
 
             // Reschedule next run
-            if (!alarm.isDisposed && updating.get() && alarm.isEmpty) {
-                alarm.addRequest({ fetchDataSafely() }, updateInterval.toMillis())
+            if (stackStatus == StatusType.DELETED) {
```

Comment:
If I press refresh and the stack has changed state, it should start updating at the new update rate (if changed)

Reply:
Added changes for after refresh polling with new update rate in the next commit